apps/chat/src/components/Chat/ChatHeader.tsx

```diff
@@ -91,6 +104,9 @@ export const ChatHeader = ({
     ConversationsSelectors.selectSelectedConversations,
   );
 
+  const isContextMenuVisible =
+    !isIsolatedView && isChatbarEnabled && !isCompareMode;
```

Comment:
I'm not sure if we need to hide context menu for compare. It's avalable in sidebar for compared conversations
![image](https://github.com/user-attachments/assets/1f100a7c-e1e4-418a-b230-25e804c45fbf)